Cargo.lock

```diff
@@ -6544,22 +7351,41 @@ version = "28.0.0"
 dependencies = [
  "aquamarine",
  "array-bytes",
- "frame-support",
- "frame-system",
- "frame-try-runtime",
+ "frame-support 28.0.0",
+ "frame-system 28.0.0",
+ "frame-try-runtime 0.34.0",
  "log",
- "pallet-balances",
- "pallet-transaction-payment",
+ "pallet-balances 28.0.0",
+ "pallet-transaction-payment 28.0.0",
  "parity-scale-codec",
  "scale-info",
  "sp-core 28.0.0",
- "sp-inherents",
+ "sp-inherents 26.0.0",
  "sp-io 30.0.0",
  "sp-runtime 31.0.1",
  "sp-tracing 16.0.0",
  "sp-version 29.0.0",
 ]
 
+[[package]]
+name = "frame-executive"
+version = "38.0.0"
```

Comment:
where are all these duplicates coming from?

Reply:
Also notices this before, funny enough this happened in a different PR I reviewed today, now here.
https://github.com/paritytech/polkadot-sdk/pull/7257#discussion_r1973048537

Will fix it.